### ISSUE TITLE
[Bug]: Error setting hashed password

### ISSUE THREAD

**Issue by IvanM9** [wg-easy/wg-easy#1555]
### Describe the bug

In the documentation for "how to generate a bcrypt hash" it says to replace each $ symbol with two $$ symbols, however, I only have access when the hash has only 1 $ symbol.

[How_to_generate_an_bcrypt_hash](https://github.com/wg-easy/wg-easy/blob/master/How_to_generate_an_bcrypt_hash.md)

**Comment by kaaax0815** [comment]
Please provide more information. E.g. your docker-compose File, OS, etc.

**Comment by IvanM9** [comment]
Hi @kaaax0815,

I used the [WireGuard template along with PiHole](https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-with-Pi-Hole) and modified the "PASSWORD" environment variable to the suggested "PASSWORD_HASH" variable and then entered the web UI.

When trying to access the web interface, placing the hashed password in the environment variable using the double $ symbol (as indicated in the documentation on "[how to hash the password](https://github.com/wg-easy/wg-easy/blob/master/How_to_generate_an_bcrypt_hash.md)"), I get an unauthorized error. However, I managed to work around it by using the generated hashed password without any modification.
